tox.ini

```diff
@@ -1,9 +1,23 @@
 [tox]
-envlist = flake8
+envlist = flake8, py27, py34
+
+[tox:travis]
+2.7 = py27
+3.4 = py34
 
 [testenv:flake8]
 deps=flake8
 # The ansible modules policy is more benevolent than standard python conding
 # conventions, therefore we probably want to silence the flake8 shouting
 # about certain errors like line length or so.
-commands=flake8 --ignore=F403,E221,E501,F405 library
+commands =
+	flake8 --ignore=F403,E221,E501,F405 library
+	flake8 tests setup.py
+
+[testenv]
+deps = mock
+       pytest
+       ansible
+       git+https://github.com/ManageIQ/manageiq-api-client-python.git
```

Comment:
For testing locally just run `tox`

As far as the `manageiq-api-client-python` is not on pip we cannot put the deps to the `setup.py`:

    error in manageiq-ansible-module setup command: 'install_requires' must be a string or list of strings containing valid project/version requirement specifiers; Invalid requirement, parse error at "'+https:/'"